main.py

```diff
@@ -0,0 +1,47 @@
+from rubikState import RubikState, Moves
```

Comment:
can you add `#!/usr/bin/python` at the beginning of the file and make it executable (`chmod a+x main.py`) so that we can directly start the program from command line with `./main.py`